lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -46,9 +65,19 @@ all_afs:
   config_get_token_append: '/^address-family (\S+) (\S+)$/'
 
 client_to_client:
-  config_get_token_append: '/^client-to-client reflection$/'
-  config_set_append: '<state> client-to-client reflection'
-  default_value: true
+  cli_nexus:
+    config_get_token_append: '/^client-to-client reflection$/'
+    config_set_append: '<state> client-to-client reflection'
+    default_value: true
+  cli_ios_xr:
+    config_set: 'router bgp <asnum>'
+    config_set_append:
+      - ' address-family <afi> <safi>'
+      - ' <state> bgp client-to-client reflection disable'
```

Comment:
My concern is that bgp_af is VRF-aware at an object level. By masking away this VRF on XR, you allow objects to conflict:

```ruby
af1 = RouterBgpAF(100, 'red', 'ipv4 unicast')
af2 = RouterBgpAF(100, 'default', 'ipv4 unicast')
af1.client_to_client = true
# because client_to_client is not VRF-aware, now af2.client_to_client is also true, unexpectedly?
```